tests/integration/suite/test_clustertemplate.py

```diff
@@ -786,6 +786,73 @@ def test_cluster_desc_update(admin_mc, list_remove_resource):
     wait_for_cluster_to_be_deleted(client, cluster.id)
 
 
+def test_update_cluster_monitoring(admin_mc, list_remove_resource):
+    cluster_template = create_cluster_template(admin_mc, [], admin_mc)
+    remove_list = [cluster_template]
+    list_remove_resource(remove_list)
+
+    tId = cluster_template.id
+    client = admin_mc.client
+    cconfig = {
+        "rancherKubernetesEngineConfig": {
+            "services": {
+                "type": "rkeConfigServices",
+                "kubeApi": {
+                    "alwaysPullImages": "false",
+                    "podSecurityPolicy": "false",
+                    "serviceNodePortRange": "30000-32767",
+                    "type": "kubeAPIService"
+                }
+            }
+        },
+        "enableClusterMonitoring": "true",
+        "defaultPodSecurityPolicyTemplateId": "restricted",
+    }
+    rev1 = client.create_cluster_template_revision(clusterConfig=cconfig,
+                                                   name="v1",
+                                                   clusterTemplateId=tId,
+                                                   enabled="true")
+    cconfig2 = {
+        "rancherKubernetesEngineConfig": {
+            "services": {
+                "type": "rkeConfigServices",
+                "kubeApi": {
+                    "alwaysPullImages": "false",
+                    "podSecurityPolicy": "false",
+                    "serviceNodePortRange": "30000-32767",
+                    "type": "kubeAPIService"
+                }
+            }
+        },
+        "enableClusterMonitoring": "false",
+        "defaultPodSecurityPolicyTemplateId": "restricted",
+    }
+    rev2 = client.create_cluster_template_revision(clusterConfig=cconfig2,
+                                                   name="v2",
+                                                   clusterTemplateId=tId,
+                                                   enabled="true")
+
```

Comment:
looks like we don't have remove_resource for rev1 and rev2?

Reply:
revisions will be deleted with the template due to owner-references